ocean_lib/ocean/test/test_ocean_compute.py

```diff
@@ -0,0 +1,537 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+from datetime import datetime
+
+from ocean_lib.assets.utils import create_checksum
+from ocean_lib.config import Config
+from ocean_lib.data_provider.data_service_provider import DataServiceProvider
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.ocean.ocean_compute import OceanCompute
+from tests.resources.ddo_helpers import (
+    get_registered_algorithm_ddo,
+    wait_for_ddo,
+    get_registered_ddo_with_compute_service,
+)
+from tests.resources.helper_functions import get_publisher_wallet
+
+
+def test_build_cluster_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+    cluster_dict = compute.build_cluster_attributes(
+        "Kubernetes", "http://10.0.0.17/my_cluster"
+    )
+    assert cluster_dict, "Cluster dictionary is None."
+    assert isinstance(cluster_dict, dict), "The cluster is not a dict."
+    assert (
+        "type" in cluster_dict and cluster_dict["type"] == "Kubernetes"
+    ), "Type does not belong to the attributes."
+    assert (
+        "url" in cluster_dict and cluster_dict["url"] == "http://10.0.0.17/my_cluster"
+    ), "Url does not belong to the attributes."
+
+
+def test_build_container_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+    container_dict = compute.build_container_attributes(
+        "node", "best_tag", "entrypoint.exe"
+    )
+    assert container_dict, "Container dictionary is None."
+    assert isinstance(container_dict, dict), "The container is not a dict."
+    assert (
+        "image" in container_dict and container_dict["image"] == "node"
+    ), "Image does not belong to the attributes."
+    assert (
+        "tag" in container_dict and container_dict["tag"] == "best_tag"
+    ), "Tag does not belong to the attributes."
+    assert (
+        "entrypoint" in container_dict
+        and container_dict["entrypoint"] == "entrypoint.exe"
+    ), "Entrypoint does not belong to the attributes."
+
+
+def test_build_server_attributes():
+    data_provider = DataServiceProvider
+    config = Ocean.config
+    compute = OceanCompute(config=config, data_provider=data_provider)
+    server_dict = compute.build_server_attributes(
+        "123", "foo_server_type", 4, 4, "20", "20", 30
```

Comment:
what are these attributes? Can you please make them more deescriptive? E.g. if you have the serverID, input "test_server_id_123" instead of "123"